src/focus.service.ts

```diff
@@ -336,10 +336,15 @@ export class FocusService {
    * Wrapper around moveFocus to dispatch arcselectingnode event
    */
   public selectNode(next: HTMLElement, scrollSpeed: number | null = this.scrollSpeed) {
+    if (!this.focusRoot.contains(next)) {
+      return;
```

Comment:
Probably good to logging a warning here

Reply:
I think this should be considered a normal functionality rather than an exception.
For instance, `arc-default-focus` selects node after it is initialized. Adding a warning here may spam the logs if focus is trapped to some other node while `arc-default-focus`ed node is initializing.

Reply:
That's fair